src/Qowaiv/Json/ModifyTypeInfo.cs

```diff
@@ -0,0 +1,38 @@
+#if NET8_0_OR_GREATER
```

Comment:
`ShouldSerialize` is supported since .NET 6

Reply:
I could not find an entry point for .NET 6. Both `JsonTypeInfo` has no way to loop through properties (at least, I could not find it), and secondly, I could not find way to register `JsonTypeInfo` modifiers at the serializer options.

Given that .NET 6.0 will not longer be supported by Microsoft in 3 days time, I think it is safe to say I would not like to invest time i this, as it would require a separate implementation.

Reply:
[https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.defaultjsontypeinforesolver.modifiers?view=net-6.0](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.defaultjsontypeinforesolver.modifiers?view=net-6.0) shows it being supported by `.NET 6`, but it mentions it is part of `System.Text.Json` `v8.0.5`, so perhaps it's only there if you target the `.NET 8` version of `System.Text.Json`.